app/assets/stylesheets/components/_user_intent_banner.scss

```diff
@@ -1,4 +1,4 @@
-.app-c-user_intent_banner {
+.app-c-user-intent-banner {
```

Comment:
This and the rename of `.app-c-user-intent-banner__title` in this file should be fixed up to the previous commit.